pysgrid/sgrid.py

```diff
@@ -351,6 +815,20 @@ def get_cell_node_lat_lon(self):
             node_lon = self.nc[node_lon_var]
         return node_lon, node_lat
 
+    def get_cell_edge1_lat_lon(self):
```

Comment:
@ocefpaf,

Naming things has never been my forte. :stuck_out_tongue_winking_eye: 

I see how that's unnecessary verbiage. I have no objections to renaming `get_cell_node_lat_lon` to `get_cell_node` for example. 

I'm inclined to suggest that method name changes be a separate PR. This PR is pretty hefty as is, and name changes can ripple into other sections of the code. It would be advantageous to not conflate the additions here with the task of more concise method names. It's also something that I can probably address in the next few weeks. Thoughts?